libraries/abstractions/platform/test/iot_test_platform_threads.c

```diff
@@ -119,23 +119,7 @@ TEST( UTIL_Platform_Threads, IotThreads_CreateDetachedThread )
 #if ( configUSE_TRACE_FACILITY == 1 )
     void threadPriorityTestFunction( void * param )
     {
-        TaskStatus_t xTaskDetails;
-
-        /* Use the handle to obtain further information about the task. */
-        vTaskGetInfo( /* The handle of the task being queried. */
-            NULL,
-
-            /* The TaskStatus_t structure to complete with information
-             * on xTask. */
-            &xTaskDetails,
-
-            /* Include the stack high water mark value in the
-             * TaskStatus_t structure. */
-            pdTRUE,
-            /* Include the task state in the TaskStatus_t structure. */
-            eInvalid );
-
-        *( int32_t * ) param = xTaskDetails.uxCurrentPriority;
+        *( int32_t * ) param = uxTaskPriorityGet( NULL );
```

Comment:
Will this be always enabled ? Checking freertos documentation: https://www.freertos.org/a00128.html INCLUDE_uxTaskPriorityGet  should be set to 1.
Previous code had a guard for configUSE_TRACE_FACILITY which was required to be enabled for `vTaskGetInfo()`

Reply:
Thanks. Good catch. It's enabled in all of our test configurations, but I've gone ahead and updated the guard.